README.md

```diff
@@ -10,7 +10,10 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 
 ```js
 
-var superagent = require('superagent-use');
+var superagent = require('superagent');
+/* Pass the superagent object to superagent-use
+   in order to be provided with new `use` functionality */
+require('superagent-use')(superagent);
```

Comment:
> it doesn't seem possible.

I'm not sure what you're saying, because before I created a new instance of superagent by using the `object-assign`.